DDG4/python/DDSim/DD4hepSimulation.py

```diff
@@ -324,6 +323,8 @@ def run(self):
 
     if self.runType == "vis":
       simple.setupUI(typ="csh", vis=True, macro=self.macroFile)
```

Comment:
```suggestion
      simple.setupUI(typ="tcsh", vis=True, macro=self.macroFile)
```

Reply:
Also in line 331, where I can't comment any more.

Reply:
Didn't know there was a difference... Changed all `csh` into `tcsh`.